content/en/flux/use-cases/gh-actions-helm-promotion.md

```diff
@@ -120,6 +126,8 @@ jobs:
       - name: Get chart version from staging
         id: staging
         run: |
+          # For OCIRepository source, use the oci-digest field instead of revision.
+          # e.g. DIGEST=$(echo ${{ github.event.client_payload.metadata.oci-digest }} | cut -d '@' -f1)
```

Comment:
I don't think this is a useful hint as the whole workflow must be different, instead of patching the HelmRelease, we need to know where the OCIRepo YAML is, and override the `ref` field to point to the digest.